var/spack/repos/builtin/packages/xyce/package.py

```diff
@@ -52,12 +52,14 @@ class Xyce(CMakePackage):
 
     variant('pymi', default=False, description='Enable Python Model Interpreter for Xyce')
     depends_on('python@3:', type=('build', 'link', 'run'), when='+pymi')
+    depends_on('py-numpy', type=('build', 'link', 'run'), when='+pymi')
+    depends_on('py-pycompadre+trilinos', type=('build', 'link', 'run'), when='+pymi')
     depends_on('py-pip', type='run', when='+pymi')
-    depends_on('py-pybind11@2.6.1:', when='+pymi')
+    depends_on('py-pybind11@2.6.1:', type=('build', 'link'), when='+pymi')
 
     # Xyce is built against an older version of Trilinos unlikely to be
     # used for any other purpose.
-    depends_on('trilinos@12.12.1 +amesos+amesos2+anasazi+aztec+basker+belos+complex+epetra+epetraext+explicit_template_instantiation+fortran+hdf5+ifpack+isorropia+kokkos+nox+sacado+suite-sparse+trilinoscouplings+zoltan+stokhos+epetraextbtf+epetraextexperimental+epetraextgraphreorderings gotype=all')
+    depends_on('trilinos@13.2.0 +amesos+amesos2+anasazi+aztec+basker+belos+complex+epetra+epetraext+explicit_template_instantiation+fortran+hdf5+ifpack+isorropia+kokkos+nox+sacado+suite-sparse+trilinoscouplings+zoltan+stokhos+epetraextbtf+epetraextexperimental+epetraextgraphreorderings gotype=long')
```

Comment:
I have a few concerns with this line:
1. `trilinos` is pinned at a specific version, which seems bad because it forces all other specs to use the same version (if they can)
2. The comment above the directive seems now outdated since 13.2.0 seems the latest version of trilinos
3. We changed the pinned version of trilinos without making it conditional to new versions. Does that mean that it was possible, in principle, to use v13.2.0 also before this PR?